### PR TITLE
Improved UI layout and added some onPress functionality. Advice needed

### PR DESCRIPTION
**Pull request by jessechong**
Created new greeter.js to further modularize the code.
1. Please view the openDailySurvey function in greeter.js
Right now, it's just pseudo code. I want the user to be able to click the "How are you feeling?" yellow text and get a form to popup. I was trying to use a modal but not sure how to implement it. Anyways, since the personal model class has an updateState method function (albeit currently empty), openDailySurvey would be passing an mcq_answer array to it. See the pseudo-code below.
```
openDailySurvey() {
    mcq_answers = array
    Get the user's current mental state ratings via a modal(?) form.
    mcq_answers.append(sadnessRating)
    mcq_answers.append(lonelinessRating)
    mcq_answers.append(anxiousnessRating)
    mcq_answers.append(stressRating)
    mcq_answers.append(angerRating)
    personalModel.updateState(mcq_answers) // array of ratings gets passed to personal model...
}
```

2. Please view _personalModelUpdate(positiveReaction) in task.js
The button, on press, will pass a boolean to this function. The issue I am having right now is I'm not sure how to interact with a task object. 
Right now, I understand that calling task.name gets a particular task's name. However, if I called task.description would I be getting a particular task's description? Likewise, if I called task.labels would I get an array of a particular task's labels that I could iterate through?
I was unable to get the server to run so I'm not sure what a "real" task looks like besides the placeholder tasks found inside the TaskList constructor (task1, task2, task3). I appreciated the help Tony and Kaj gave me on Discord but their advice does not apply to Windows. Would appreciate it if Matt (@mmarano25) could write a "How to start server on local machine on Windows for dummies" tutorial somewhere.

### CODE REVIEW

File: mesa/README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# MESA
\ No newline at end of file
+# MESA
+
+# FRONT-END DEPENDENCIES:
```
Comment (kajdreef) [inline mesa/README.md]: Change: `# FRONT-END DEPENDENCIES:`
To: `## FRONT-END DEPENDENCIES:`

File: mesa/README.md [hunk 2]
```diff
@@ -1 +1,4 @@
...
+# react-native-elements
```
Comment (kajdreef) [inline mesa/README.md]: You probably want to create a list here, which is done using:
`+` for an unordered list and just numbers `1.` for a ordered list. 

If it is a header you also want to add `##` before it so we keep an structure in the different sections.

File: mesa/README.md (hunk 1 above)
Comment (jessechong) [reply mesa/README.md]: I will delete this line actually. I forgot that all of the dependencies I install go into "node_modules" which is then available to everyone when they merge the branch. So there's no need for this line.

**Comment by jessechong** [comment]
Thanks @kajdreef.
I don't know why I assumed everything was already implemented.

**Comment by jessechong** [comment]
@kajdreef 
> but maybe we first need a good task list to determine what information we will be sending back.
Don't we already have `tasks.json`? 
Also, I think that the server should be sending back the task title, task description, and task labels.

**Comment by kajdreef** [comment]
> Don't we already have `tasks.json`?

Yeah, but I only added three tasks in there. And I don't know if those are really well done. It was more dummy data so that we can actually return something from the server.

**Pull request merged**